sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -1175,4 +1190,94 @@ public void executeBundles(ProcessContext context) {
       }
     }
   }
+
+  /** Export FHIR resources from a FHIR store to new line delimited json files on GCS. */
+  public static class ExportGcs extends PTransform<PBegin, ExportGcs.Result> {
```

Comment:
We might want to add some tests for this. You can find a few examples added by Jacob previously.

Reply:
Tests added. But I don't have permission on the testing project to run it. In the meantime, I wrote another script to test on my own project (not in this commit but identical to the test) and it worked fine.